sqlglot/parser.py

```diff
@@ -2984,6 +2984,14 @@ def _parse_select(
                     if table
                     else self._parse_select(nested=True, parse_set_operation=False)
                 )
+
+                # Transform exp.Values into a exp.Table to pass through parse_query_modifiers
+                # in case a modifier (e.g. join) is following
+                if table and isinstance(this, exp.Values):
+                    alias = this.args.get("alias")
+                    this.set("alias", None)
+                    this = exp.Table(this=this, alias=alias)
```

Comment:
will this have double aliases? because Values has an alias, and now table?

Reply:
should the alias get popped off?

Reply:
It will get popped off in the line `this.set("alias", None)` right before it's wrapped; I went with this approach as the `alias` is not guaranteed to be there, so `alias.pop()` would require a branch